cves/kernel/CVE-2020-11668.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: It was discovered that a few media drivers in the Linux software were not properly validating USB device descriptors that provide information to the host, such as the version of the USB supported by the device, who made the device, etc. The vulnerability this created was if a physically present user with a specially constructed USB device could use it to cause a denial-of-service (crash) or possibly for privilege escalation.
```

Comment:
I would re-word this sentence a little bit. Right now it's worded a little bit confusingly.

"The vulnerability this created was if a physically present user with a specially constructed USB device could use it to cause a denial-of-service (crash) or possibly for privilege escalation."

Reply:
maybe just punctuation idk